app/src/js/components/ReconciliationReports/gnf-report.js

```diff
@@ -0,0 +1,117 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import groupBy from 'lodash/groupBy';
+import {
+  searchReconciliationReport,
+  clearReconciliationSearch,
+} from '../../actions';
+import SortableTable from '../SortableTable/SortableTable';
+import Search from '../Search/search';
+import ReportHeading from './report-heading';
+import { handleDownloadJsonClick } from '../../utils/download-file';
+import { tableColumnsGnf } from '../../utils/table-config/reconciliation-reports';
+import { getFilesSummary, getGranuleFilesSummary } from './reshape-report';
+
+const GnfReport = ({
+  dispatch,
+  filterString,
+  recordData,
+  reportName,
+}) => {
+  const {
+    filesInCumulus,
+    granulesInCumulusCmr,
+    filesInCumulusCmr,
+    reportStartTime = null,
+    reportEndTime = null,
+    error = null
+  } = recordData || {};
+
+  const { filesInDynamoDb } = getFilesSummary(filesInCumulus);
+  const { granuleFilesOnlyInCumulus, granuleFilesOnlyInCmr } = getGranuleFilesSummary(filesInCumulusCmr);
+
+  const { onlyInCmr = [], onlyInCumulus = [] } = granulesInCumulusCmr;
+
+  const cmrGranules = onlyInCmr.map((granule) => {
+    const { GranuleUR } = granule;
+    return {
+      granuleId: GranuleUR,
+      cmr: true,
+      cumulus: false,
+      s3: false,
+    };
```

Comment:
We could get collectionId here, construct it from `shortName` and `Version`.

Reply:
https://github.com/nasa/cumulus-dashboard/pull/840/commits/0cb97ec11a56d4928ab1176f9212dd7fe61b2b00